### ISSUE TITLE
loss functions for fine-tuning embedding and reranker models

### ISSUE THREAD

**Issue by hank317** [FlagOpen/FlagEmbedding#1444]
May I ask whether the BGE-M3-embedding and BGE-reranker-MiniCPM models use **InfoNCE loss** during fine-tuning?

From the code implementation, it appears that **cross-entropy loss** is used. Can I interpret this as using a cross-entropy formulation while still being essentially equivalent to **InfoNCE** in principle?

**Comment by 545999961** [comment]
Yes, you are right, this is equivalent to the InfoNCE loss.
